src/components/PurchaseOrder/PO.js

```diff
@@ -50,11 +50,13 @@ import { RenewalsView } from './renewals';
 import LinesLimit from './LinesLimit';
 import POInvoicesContainer from './POInvoices';
 import {
+  isOpenAvailableForOrder,
   isReceiveAvailableForOrder,
 } from './util';
 import { UpdateOrderErrorModal } from './UpdateOrderErrorModal';
 
 import css from './PO.css';
+import { getOrderApprovalsSetting } from '../../common/utils/getOrderApprovalsSetting';
```

Comment:
shouldn't it be above, around `getAddresses` imported `from '../../common/utils';`